detox/local-cli/init.js

```diff
@@ -76,7 +76,7 @@ function createDefaultConfigurations() {
         config: 'e2e/jest.config.js',
       },
       jest: {
-        setupTimeout: 120000,
+        setupTimeout: 300000,
```

Comment:
@remarkablemark thanks for the contribution! 
@noomorph we need to decide whether we should replace the default value in the docs or here.

@remarkablemark - in general, you should be able to configure the test-runner and set a jest runner with five minutes setup-timeout or whatever is needed for your project.

Reply:
Thanks for the response. Happy to close the PR if the docs get updated.

Reply:
@remarkablemark thanks for your keen eye! 👍 Yes, we updated the docs.

If you have other suggestions, feel welcome to create an issue or a pull request.